manim/mobject/mobject.py

```diff
@@ -557,47 +629,117 @@ def update(self, dt=0, recursive=True):
                 submob.update(dt, recursive)
         return self
 
-    def get_time_based_updaters(self):
+    def get_time_based_updaters(self) -> "list[Callable]":
+        """Return all updaters using the ``dt`` parameter.
+
+        The updaters use this parameter as the input for difference in time.
+
+        Returns
+        -------
+        List[:class:`Callable`]
+            The list of time based updaters.
+
+        See Also
+        --------
+        :meth:`get_updaters`
+        :meth:`has_time_based_updater`
+
+        """
         return [updater for updater in self.updaters if "dt" in get_parameters(updater)]
 
-    def has_time_based_updater(self):
+    def has_time_based_updater(self) -> bool:
+        """Test if ``self`` has a time based updater.
+
+        Returns
+        -------
+        class:`bool`
+            ``True`` if at least one updater uses the ``dt`` parameter, ``False`` otherwise.
+
+        See Also
+        --------
+        :meth:`get_time_based_updaters`
+
+        """
         for updater in self.updaters:
             if "dt" in get_parameters(updater):
                 return True
         return False
 
-    def get_updaters(self):
+    def get_updaters(self) -> "list[Callable]":
```

Comment:
add `from typing import List` and then
```suggestion
    def get_updaters(self) -> List[Callable]:
```